projects/avodata/firstStage.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+import math
+
+
+def factorial(n):
+    return int(math.gamma(n + 1))
+
+
+def n_choose_k(n, k):
+    return factorial(n) / (factorial(k) * factorial(n - k))
+
+
+def generate_synthetic_a(num_of_professions):
+    num_of_rows = int(n_choose_k(num_of_professions, 2))
+    A = np.zeros(shape=(num_of_rows, num_of_professions), dtype=np.float32)
+    row = 0
+
+    for i in range(num_of_professions):
+        for j in range(i + 1, num_of_professions):
+            A[row][i] = 1
+            A[row][j] = -1
+            row += 1
+
+    return A
+
+
+def generate_synthetic_c(m, i, a):
+    c = np.dot(a, m[i].transpose())
```

Comment:
You can return directly, without an intermediate `c` variable.

Same for `solve`.

The name of the method already has the information of what is returned.

Reply:
done